actions/run.py

```diff
@@ -0,0 +1,12 @@
+from lib.base import BaseJiraAction
+
+__all__ = [
+    'ActionManager'
+]
+
+
+class ActionManager(BaseJiraAction):
+
+    def run(self, **kwargs):
+        action = kwargs.pop('action')
+        return getattr(self, action)(**kwargs)
```

Comment:
I think it's better to specify `self._client` to the first argument of `getattr` in case of actions that simply call method of JIRA instance like this action. By this changes, you can omit the implementation you've changed at `actions/lib/base.py`.

And it's better to add exception handler processing for assumed exceptions like following.
* AttributeError: `getattr` may raise (c.f. https://docs.python.org/3.6/library/functions.html#getattr).
* JIRAError: each methods of JIRA instance may raise.